src/modes/include/hardware/lamp_type.hpp

```diff
@@ -471,6 +471,30 @@ struct LampTy
     }
   }
 
+  /**
+   * \brief Fill the display with some colors, with an optional cutoff value
+   * \param[in] PaletteTy an array of several colors
+   * \param[in] cutOff between 0 and 1, how much this PaletteTy will fill the
+   * display before suddently cutting off
+   */
+  void LMBD_INLINE fill(const modes::colors::PaletteTy& palette, const float cutOff = 1)
```

Comment:
not the use for fill, it should fill the strip with the same color
Maybe a fill_palette function would be a better name ?